.github/workflows/release-workflow.yml

```diff
@@ -0,0 +1,66 @@
+name: Release Workflow
+run-name: ${{ format('{0} triggered {1} on {2}', (github.event_name == 'workflow_dispatch' && format('User {0}', github.actor) || format('{0} event', github.event_name) ), github.workflow, github.ref) }}
```

Comment:
If I set up the CI/CD bot with an endpoint would it be possible for this to notify it directly?